client/src/components/DashboardCoponent/Content.tsx

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React, { useState } from 'react';
+import Card from './ProfileCard';
+import { Grid, Button, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles(() => ({
+  bottomButton: {
+    margin: 'auto',
+  },
+  btnContainer: {
+    padding: '2rem 0',
+    display: 'grid',
+    width: '100%',
+  },
+}));
+
+function Content() {
+  const [length, setLength] = useState(6);
+
+  const classes = useStyles();
+
+  const toggelLengthHandler = () => {
+    if (length === 6) {
+      setLength(12);
+    } else {
+      setLength(6);
+    }
+  };
+
+  return (
+    <>
+      <Grid container spacing={4}>
+        {length === 6 ? (
```

Comment:
As opposed to hard coding 6/12 cards, maybe you can have an array with 6/12 elements and map over it to display cards.
Eg:
 `const profiles = [1, 2, 3, 4, 5, 6]

profiles.map((profile,idx) => (
<Card />
))
`